src/Microsoft.Extensions.DependencyModel/DependencyContext.cs

```diff
@@ -15,13 +15,46 @@ public class DependencyContext
 
         private static readonly Lazy<DependencyContext> _defaultContext = new Lazy<DependencyContext>(LoadDefault);
 
-        public DependencyContext(string target, string runtime, CompilationOptions compilationOptions, CompilationLibrary[] compileLibraries, RuntimeLibrary[] runtimeLibraries)
+        public DependencyContext(string target,
+            string runtime,
+            bool isPortable,
+            CompilationOptions compilationOptions,
+            CompilationLibrary[] compileLibraries,
+            RuntimeLibrary[] runtimeLibraries,
+            IReadOnlyList<KeyValuePair<string, string[]>> runtimeGraph)
```

Comment:
I generally prefer taking in `IEnumerable` in these situations for the most flexibility. Personal preference mostly though, so if there are good reasons to leave it as `IReadOnlyList` that's OK too.